extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -470,19 +471,21 @@ protected function getSubject() {
 	}
 
 	/**
-	 * @template founderTips
+	 * @template digestLayout
 	 */
 	public function body() {
 		$this->response->setData( [
 			'salutation' => $this->getSalutation(),
 			'summary' => $this->getMessage( 'emailext-founder-newly-created-summary', $this->wikiUrl, $this->wikiName )->parse(),
 			'extendedSummary' => $this->getMessage( 'emailext-founder-newly-created-summary-extended' )->text(),
 			'details' => $this->getDetailsList(),
+			'hasContentFooterMessages' => true,
 			'contentFooterMessages' => [
 				$this->getMessage( 'emailext-founder-visit-community', $this->wikiUrl, $this->wikiName )->parse(),
-				$this->getMessage( 'emailext-founder-happy-wikia-building' )->text(),
-				$this->getMessage( 'emailext-emailconfirmation-community-team' )->text(),
+				$this->getMessage( 'emailext-founder-happy-wikia-building' )->text()
 			],
+			'signatureIcon' => $this->findSignatureIcon(),
+			'signature' => $this->createEmailSignature()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "signature" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)